### PR TITLE
Added ability for users to add markers to map. Removed location field…

### PR DESCRIPTION
**Pull request by arianazhu**
… from comments

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/data/Marker.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package com.google.sps.data;
+
+/** This class represents a marker on the homepage map. */
+public class Marker {
+
+    private double lat;
+
+    private double lng;
+
+    private String content;
```
Comment (taruss2011) [inline portfolio/src/main/java/com/google/sps/data/Marker.java:10]: Nit: I would not put in the extra blank line before these 3 variables.
But I don't insist.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -35,6 +40,97 @@ function addRandomQuote() {
   quoteContainer.innerText = quote;
 }
 
+/** Creates a map and adds it to the page. */
+function createMap() {
+    var centerUSALatLng = {lat: 39.540997, lng: -98.852047};
```
Comment (taruss2011) [inline portfolio/src/main/webapp/script.js:45]: var => const ?

Comment (taruss2011) [inline portfolio/src/main/webapp/script.js:45]: Great naming of the variable. Eliminates a need for a comment.

File: portfolio/src/main/java/com/google/sps/servlets/MarkerServlet.java [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+        response.getWriter().println(json);
+    }
+
+    /** Accepts a POST request containing a new marker. */
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) {
+        double lat = Double.parseDouble(request.getParameter("lat"));
+        double lng = Double.parseDouble(request.getParameter("lng"));
+        String content = Jsoup.clean(request.getParameter("content"), Whitelist.none());
```
Comment (taruss2011) [inline portfolio/src/main/java/com/google/sps/servlets/MarkerServlet.java:41]: Great. Where did you hear about Jsoup?

**Review by taruss2011 — APPROVED** [review]
Great change.
Minor nits only to look at.

File: portfolio/src/main/java/com/google/sps/servlets/MarkerServlet.java (hunk 3 above)
Comment (arianazhu) [reply portfolio/src/main/java/com/google/sps/servlets/MarkerServlet.java:41]: The walkthrough provided us an example using Jsoup, which I followed for the map marker portion!

**Pull request merged**